### PR TITLE
Add sig services

### PR DESCRIPTION
**Pull request by tumido**
Resolves: #22 

~MTG Doodle link: https://doodle.com/poll/s8qgpy5n92tapw9f (WIP because awaiting decision here)~

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -2,6 +2,8 @@ aliases:
   sig-data-science-leads:
     - erikerlandson
     - michaelclifford
+  sig-services-leads:
+    - tumido
```
Comment (durandom) [inline OWNERS_ALIASES:6]: any other volunteers?

File: sig-data-science/README.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ The [charter](charter.md) defines the scope and governance of the Data Science S
 
 ## Meetings
 * Data Science SIG meeting: [Wednesdays at 10:00 ET (East Time)](meet.google.com/ufs-hgvi-oni) (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=ET%20%28East%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1KecKMMva2wQxUZFdBpd291q75Z7ATp7F1YihMbuC_xg/edit).
```
Comment (durandom) [inline sig-data-science/README.md:19]: I guess that popped in because you ran the generator.

File: sig-services/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Contact
+- [Mailing list]()
+- [Open Community Issues/PRs](https://github.com/open-services-group/community/labels/sig%2Fservices)
+- Steering Committee Liaison: Tom Coufal (**[@tumido](https://github.com/tumido)**)
+
+## Subprojects
+
+The following [subprojects][subproject-definition] are owned by sig-services:
+### AICoE-CD
```
Comment (durandom) [inline sig-services/README.md:35]: I'm ok with the list of subprojects for now, though this'll probably change after the first few sig meetings

**Review by durandom — COMMENTED** [review]
/lgtm

File: OWNERS_ALIASES (hunk 1 above)
Comment (tumido) [reply OWNERS_ALIASES:6]: Let's discuss than during the first meeting along with the subprojects. For now, we should merge this as is, wdyt?

**Pull request merged**